app/assets/stylesheets/generic/timeline.scss

```diff
@@ -0,0 +1,150 @@
+.timeline {
+  list-style: none;
+  padding: 20px 0 20px;
+  position: relative;
+}
+
+.timeline:before {
+  top: 0;
+  bottom: 0;
+  position: absolute;
+  content: " ";
+  width: 3px;
+  background-color: #eeeeee;
+  left: 40px;
+  margin-left: -1.5px;
+}
+
+.timeline > li {
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.timeline > li:before,
+.timeline > li:after {
+  content: " ";
+  display: table;
+}
+
+.timeline > li:after {
+    clear: both;
+}
+
+.timeline > li > .timeline-panel {
+  width: 80%;
+  float: left;
+  border: 1px solid #d4d4d4;
```

Comment:
Color literals like `#d4d4d4` should only be used in variable declarations; they should be referred to via variable everywhere else.